src/test/java/com/synopsys/integration/alert/audit/controller/AuditEntryControllerTestIT.java

```diff
@@ -94,6 +94,12 @@ public void setup() throws AlertDatabaseConstraintException {
 
     @AfterEach
     public void cleanup() throws AlertDatabaseConstraintException {
+        auditEntryRepository.flush();
+        descriptorConfigRepository.flush();
+        fieldValueRepository.flush();
+        auditNotificationRepository.flush();
+        notificationRepository.flush();
```

Comment:
I wonder if there's a way to just get the transaction manager or persistence manager and flush at the top level.

Reply:
I think there is a different way to define and setup testcontainers with the tests that need a database to provide a clean instance with each test.  I would like to revisit this because I think we can do better.

Reply:
There is, but the performance hit is extremely costly. We do it in `jira-service` integration tests and it adds significant overhead.